test/cypress/integration/components/filter-blog-posts.js

```diff
@@ -0,0 +1,300 @@
+    // Then the list of topics should show only tags that contain the word "mortgage"
+    page.resultsContent().should( 'contain', 'mortgage' );
+    // And when I select a topic in the list
+    filter.clickTopic( 'Servicemembers' );
+    // And I click "Apply filters" button
+    page.applyFilters();
+    // Then I should see only results tagged with the selected topic
+    page.notification().should( 'be.visible' );
+    page.resultsContent().should( 'contain', 'service' );
+    // And the page url should contain "topics=mortgages"
+    cy.url().should( 'include', 'topics=mortgages' );
+    // And the page url should contain "topics=servicemembers"
+    cy.url().should( 'include', 'topics=servicemembers' );
+  } );
+  it( 'Select category and topic', () => {
+    // When I select a checkbox in the Category list
+    filter.checkCategoryId( 'policy_compliance' );
+    // When I select a checkbox in the Topic list
+    filter.clickTopic( 'Students' );
+    // And I click "Apply filters" button
+    page.applyFilters();
+    // Then I should see only results that are both in the selected category and tagged with the selected topic
+    page.notification().should( 'be.visible' );
+    page.resultsContent().should( 'contain', 'student' );
+    page.resultsContent().should( 'contain', 'policy' );
+    // And the page url should contain "categories=policy_compliance"
+    cy.url().should( 'include', 'categories=policy_compliance' );
+    // And the page url should contain "topics=students"
+    cy.url().should( 'include', 'topics=students' );
+  } );
+  it( 'Clear and hide filters', () => {
+    // When I select the last checkbox in the Category list
+    filter.checkCategoryName( 'Info for consumers' );
+    // When I select a checkbox in the Topic list
+    filter.clickTopic( 'Consumer complaints' );
+    // And I click "Apply filters" button
+    page.applyFilters();
+    // Then the page url should contain "categories=info-for-consumers"
+    cy.url().should( 'include', 'categories=info-for-consumers' );
+    // And the page url should contain "topics=consumer-complaints"
+    cy.url().should( 'include', 'topics=consumer-complaints' );
+    // Then I should see only results that are both in the selected category and tagged with the selected topic
+    page.resultsContent().should( 'contain', 'consumer' );
+    // And when I click "Show filters"
+    page.showFilters();
+    // And when I click "Clear filters"
+    page.clearFilters();
+    // And I click "Hide filters" button
+    filter.hide();
+    // Then the page url should not contain "categories=info-for-consumers"
+    cy.url().should( 'not.include', 'categories=info-for-consumers' );
+    // And the page url should not contain "topics=consumer-complaints"
+    cy.url().should( 'not.include', 'topics=consumer-complaints' );
+    // And when I click "Apply filters"
+    page.applyFilters();
+    // Then I should see the full list of results
+    page.notification().should( 'be.visible' );
```

Comment:
```suggestion
    page.notification().should( 'not.be.visible' );
```

Reply:
When I hide the filter the page notification is not visible.